helm/defectdojo/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.7.1"
+appVersion: "1.8.0"
 description: A Helm chart for Kubernetes to installs DefectDojo
 name: defectdojo
```

Comment:
Sorry for splitting hair but I would do 1.4.0 (feature added, not bugfix)

Reply:
The line you pointed out was not the Chart version, but I understood it as such. Pushed a new commit.